core/templates/dev/head/pages/library-page/library-page.directive.ts

```diff
@@ -48,17 +47,15 @@ angular.module('oppia').directive('libraryPage', [
       controllerAs: '$ctrl',
       controller: [
         '$http', '$log', '$rootScope', '$scope', '$timeout', '$uibModal',
-        '$window', 'AlertsService', 'ConstructTranslationIdsService',
-        'LearnerDashboardActivityIdsObjectFactory',
```

Comment:
Out of curiosity, why was this removed?

Reply:
It was not being used in this directive.

Reply:
yup